apps/web/src/app/challenge/[slug]/right-wrapper.tsx

```diff
@@ -52,7 +56,7 @@ export function RightWrapper({ track, challenge }: Props) {
           isSuccessful,
         });
 
-        return handleSuccessfulSubmission(submission.isSuccessful, submission.id, track?.slug);
+        return await handleSuccessfulSubmission(submission.isSuccessful, submission.id, track?.slug);
```

Comment:
this was a latent bug?

Reply:
no, its awaited where the function is called

Reply:
removed, unneeded.